img/v/virginplus.ca.svg

```diff
@@ -0,0 +1 @@
-1.473 54.829-1.458 56.503.589 63.405 2.121 7.151 5.095 10.125 12.191 12.188 5.193 1.51 5.478 1.485 10.764-.935 3.935-1.802 6.157-3.637 8.069-6.667 1.452-2.299 2.423-4.396 2.159-4.661-.265-.264-.11-1.773.343-3.353.602-2.099.425-3.203-.656-4.101-1.377-1.143-1.325-3.175.184-7.107.299-.78-.627-2.589-2.059-4.02-2.524-2.525-2.554-2.697-.991-5.72 1.368-2.646 1.423-3.679.36-6.823-.689-2.038-1.723-4.156-2.299-4.706-.575-.55-1.08-4.15-1.121-8-.041-3.85-.558-7.9-1.149-9-1.106-2.058-1.029-33.559.223-91.5l.616-28.5 3.245-3c1.785-1.65 5.022-4.348 7.193-5.995 6.018-4.567 38.388-25.218 52.447-33.46 6.875-4.031 14.178-8.39 16.229-9.687 2.05-1.297 3.898-2.358 4.106-2.358.208 0 4.38-2.311 9.272-5.136 4.891-2.825 12.493-7.056 16.893-9.401 4.4-2.346 13.175-7.042 19.5-10.437 10.229-5.489 28.791-14.673 39.229-19.408 8.185-3.714 30.76-11.618 33.18-11.618.513 0 3.854-1.423 7.424-3.162 7.653-3.729 13.328-9.076 16.266-15.326 2.293-4.878 2.107-7.127-1.011-12.175-1.767-2.862-2.569-3.229-8.407-3.855-3.55-.38-7.925-.368-9.722.027-1.797.395-9.708 4.042-17.581 8.104-7.872 4.063-15.383 7.387-16.691 7.387-2.902 0-5.393-2.599-6.816-7.112-1.364-4.325-2.062-33.722-1.069-45.016.417-4.746.367-10.141-.112-11.99-.537-2.07-.492-3.97.117-4.945 1.139-1.823.217-7.745-2.02-12.983-2.597-6.081-9.788-9.02-17.287-7.064m-99.998 44.685c-7.213 3.761-11.228 10.348-12.558 20.604-2.198 16.937-6.078 56.163-6.61 66.821-.536 10.737-.438 11.623 1.485 13.35 1.194 1.072 3.565 1.85 5.641 1.85 4.994 0 14.309-5.725 17.82-10.952 2.584-3.849 2.828-5.058 3.213-15.956.23-6.486.62-14.267.867-17.292.247-3.025.476-10.907.508-17.514.033-6.608.76-16.677 1.615-22.375 1.264-8.414 1.327-11.041.333-13.986-2.108-6.244-6.178-7.749-12.314-4.55M411.5 259.017c-4.485.969-8.54 3.374-13 7.709-4.816 4.682-8.057 12.679-6.644 16.396 2.464 6.48 8.165 6.962 16.547 1.399 10.235-6.794 13.513-13.725 10.534-22.272-1.186-3.401-3.002-4.19-7.437-3.232m56 33.574c-4.4 1.962-11.15 4.24-15 5.063-7.933 1.695-14.441 5.081-16.858 8.77-3.879 5.919-4.723 10.985-6.177 37.076-.307 5.5-.943 12.475-1.413 15.5-.471 3.025-.908 9.55-.971 14.5-.063 4.95-.384 10.915-.712 13.256-1.689 12.025 2.591 20.999 9.481 19.881 8.728-1.416 17.448-8.794 19.229-16.268.507-2.128 1.371-12.644 1.92-23.369.55-10.725 1.418-21.3 1.93-23.5 2.771-11.906 3.678-14.403 6.803-18.734 1.878-2.604 4.845-5.742 6.592-6.972 1.747-1.23 4.296-3.262 5.666-4.516 1.369-1.253 2.795-2.278 3.168-2.278 1.448 0 4.903-6.993 5.022-10.166.232-6.161-4.357-11.875-9.495-11.822-.652.007-4.785 1.617-9.185 3.579m-76.249 32.438c-9.459 5.425-12.925 13.274-14.34 32.471-.526 7.15-1.422 17.725-1.989 23.5-.567 5.775-1.28 19.5-1.583 30.5-.537 19.465-.494 20.06 1.594 22.25 3.027 3.176 7.68 2.917 13.121-.729 11.868-7.952 11.389-7.47 11.544-11.617.393-10.538 2.51-34.363 3.368-37.904.533-2.2 1.249-8.05 1.591-13 .342-4.95 1.268-12.629 2.058-17.065 1.866-10.476 1.374-25.276-.957-28.835-2.39-3.646-7.569-3.492-14.407.429M337.169 651.75c94.968.141 250.668.141 346 0s17.631-.256-172.669-.256-268.299.115-173.331.256"/></g></svg>
```

Comment:
This SVG looks like it's been run through a raster to vector image ML program and I've never found any such program producing decent results.
As there doesn't seem to be a proper SVG for Virgin Plus please replace it with a PNG. A+ for searching for a SVG file tho! :+1:

Reply:
I have updated the Virgin Plus logo with a PNG, but the file is still 3.7 kB, even after running it through TinyPNG.